nvchecker/core.py

```diff
@@ -412,8 +413,9 @@ async def process_result(
       check_version_update(oldvers, r1, verbose)
       entry_waiter.set_result(r1.name, r1.version)
       ret[r1.name] = r1.version
+      results.append(r1)
```

Comment:
What about returning data like this:

```suggestion
      ret[r1.name] = r1
```

In this way we can avoid returning duplicate information in two ways.